app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt

```diff
@@ -85,103 +92,24 @@ fun HomeScreenContent(title: String) {
                 .size(dimensionResource(id = R.dimen.margin_padding_size_xxlarge))
         )
         Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.margin_padding_size_medium)))
-        SearchBar()
-        Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.margin_padding_size_medium)))
-
-        Column(
-            modifier = Modifier
-                .fillMaxSize()
-                .verticalScroll(rememberScrollState())
+        AnimatedVisibility(
+            visible = searchBarVisible,
+            enter = slideInVertically { slideOffset } + expandVertically(expandFrom = Alignment.Top) + fadeIn(),
+            exit = slideOutVertically() + shrinkVertically() + fadeOut()
         ) {
-            TextBoxCarreras(text = "Licenciatura en Ing en Sistemas", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Informática", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Industrial", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Electrónica", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Sistemas", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Informática", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Industrial", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Electrónica", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Sistemas", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Informática", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Industrial", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Electrónica", onClick = {})
-            TextBoxCarreras(text = "LICENCIATURA EN BIOLOGÍA", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Informática", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Industrial", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Electrónica", onClick = {})
+            val subjects = listOf(
```

Comment:
move this list to the definition of the class , and get it from there, it can be a file constant , but avoid defining mocked list or any kind of data in the UI , composable are only to handle states, not to define data(even mocked)